web/config/initializers/omniauth.rb

```diff
@@ -0,0 +1,3 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :github, ENV['GITHUB_OAUTH_CLIENT_ID'], ENV['GITHUB_OAUTH_CLIENT_SECRET'], scope: 'user,repo'
```

Comment:
Can you add these 2 env vars to the README and indicate that a user should have them set in order to try out the OAuth to GH?